deps/wazuh_testing/wazuh_testing/scripts/check_files.py

```diff
@@ -152,8 +153,15 @@ def get_data_information(item):
     mode = mode_str[-3:] if len(mode_str) > 3 else mode_str
     _type = 'directory' if os.path.isdir(item) else 'file'
     protection = get_filemode(stat_info.st_mode)
+    last_update = datetime.fromtimestamp(os.path.getmtime(item)).strftime('%Y-%m-%d %H:%M:%S')
+    if _type != 'directory':
+        checksum = hashlib.md5(open(item, 'rb').read()).hexdigest()
 
-    return {'type': _type, 'user': user, 'group': group, 'mode': mode, 'prot': protection}
+        return {'type': _type, 'user': user, 'group': group, 'mode': mode, 'permission': protection,
+                'last_update': last_update, 'checksum': checksum}
+    else:
+        return {'type': _type, 'user': user, 'group': group, 'mode': mode, 'permission': protection,
```

Comment:
```suggestion
        return {'type': _type, 'user': user, 'group': group, 'mode': mode, 'permissions': protection,
```

Reply:
https://github.com/wazuh/wazuh-qa/commit/783bf029455d1fe7357cb05132b9b76de5879f59 solves it